source/transfer_list.rst

```diff
@@ -141,15 +141,39 @@ The TE header is defined in :numref:`tab_te_header`.
      - The size of this entry header in bytes. This field is set to 8 for the TE header layout described in this version of the table.
 
 
-Entry type ranges
+Entry type allocation
 -----------------
 
-The content of the data section is determined by the tag id. The tag id space contains two ranges:
-
- #. Standard range, and
- #. Non-standard range
-
-The *tag_id* ranges are described in :numref:`tab_tag_id_ranges`.
+Tag IDs must be allocated in this specification before use. A new tag ID can be
+allocated by submitting a pull request to this repository that adds a description
+of the respective TE data layout to this specification. Tag IDs do not have to be
+allocated in order. Submitters are encouraged to try to group tag IDs together in
+logical clusters (e.g. all tags related to a particular chipset or to a
```

Comment:
> particular chipset

I think we should drop that bit. Ideally things should be common to all chipsets, at least for a particular architecture (x86, ARM, etc.)

Definitely firmware projects may have their own things

Reply:
This is OK (I'm a bit dubious about vendors though). I don't think we need such a large space between things. Let's take a guess at how many things might be needed, so the tags are somewhat close toghet.